microsoft/botframework-solutions
# [Enterprise Bot][TypeScript] Move tslint.json to enterprise-bot folder

## Description

- Move `tslint.json` file with Microsoft rules into _enterprise-bot_ folder
	
## Related Issue
~-~
## Testing Steps
1. Open `AI/templates/Enterprise-Template/src/typescript/enterprise-bot`
2. Run  `npm i` (to install tslint dependency)
3. Run `npm run build` and check that the execution of build script was successful
	
## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
